ui/src/dashboards/components/TableOptions.tsx

```diff
@@ -36,91 +36,28 @@ interface Options {
   fixFirstColumn: boolean
 }
 
+interface QueryConfig {
```

Comment:
there's another interface QueryConfig in `ui/src/types/query.ts` maybe this should have a different name? Or should it be the same as that? Is this a slimmed down version of the other query config?

Reply:
this interface was a slimmed down version that only included the used parts of the interface. I didn't realize that type already exists. I'll change it to use the interface in query.ts.